reference/reference.md

```diff
@@ -1,5 +1,12 @@
 # References
 
+This section gives a more complete overview of several things that are available in Valtimo like:
```

Comment:
Things feels really vague here, but I'm having a hard time coming up with an alternative...

Reply:
I wasn't very happy with this either, but i was struggling to think of something good.

Reply:
'This section provides an index to features and functionality within Valtimo'?